go.mod

```diff
@@ -7,7 +7,6 @@ require (
 	github.com/hashicorp/go-hclog v1.2.0
 	github.com/hashicorp/hcl/v2 v2.8.0
 	github.com/hashicorp/logutils v1.0.0
-	github.com/minamijoyo/tfschema v0.6.0
```

Comment:
this library isn't compatible with newer terraform versions. Therefore I couldn't use it anymore.